src/OpenTelemetry.Exporter.InMemory/InMemoryExporterMetricsExtensions.cs

```diff
@@ -147,11 +147,13 @@ public static MeterProviderBuilder AddInMemoryExporter(
         });
     }
 
-    private static MetricReader BuildInMemoryExporterMetricReader(
+    private static PeriodicExportingMetricReader BuildInMemoryExporterMetricReader(
         ICollection<Metric> exportedItems,
         MetricReaderOptions metricReaderOptions)
     {
+#pragma warning disable CA2000
```

Comment:
nit: i would prefer to see comments on all the #pragma disable explaining what is being disabled.